nuxeo-vision-core/src/main/java/org/nuxeo/vision/core/service/Vision.java

```diff
@@ -37,44 +37,44 @@
 
     /**
      * @param blob the image blob
-     * @param features the feature to request from the service
+     * @param feature(s) the feature to request from the service
```

Comment:
The param in javadoc should have the same naming than the parameter (for html links).